src/test/features/claimant-response/helpers/taskListBuilder.ts

```diff
@@ -24,13 +24,14 @@ describe('Claimant response task list builder', () => {
   let claimantResponseDraft: DraftClaimantResponse
 
   beforeEach(() => {
-    claim = new Claim().deserialize({ ...claimStoreServiceMock.sampleClaimObj, ...claimStoreServiceMock.sampleDefendantResponseObj })
+    claim = new Claim().deserialize({ ...claimStoreServiceMock.sampleClaimObj, ...claimStoreServiceMock.sampleDefendantResponseObj, ...claimStoreServiceMock })
```

Comment:
do we need `...claimStoreServiceMock` ? I think otherwise

Reply:
Definitely not, that was stupid, I have no idea how that made the code and how that disappeared. Magic.